### PR TITLE
Update to core 8 alpha 940

### PR DESCRIPTION
**Pull request by andreasohlund**
Removed the samples project since I couldn't figure out why we would need it

### CODE REVIEW

**Review by DavidBoike — APPROVED** [review]

File: src/NServiceBus.Heartbeat/HeartbeatSender.cs [hunk 1]
```diff
@@ -23,41 +22,56 @@ class HeartbeatSender : FeatureStartupTask, IDisposable
...
+            stopSendingHeartbeatsTokenSource = new CancellationTokenSource();
 
-            return Task.FromResult(0);
-        }
-
-        protected override Task OnStop(IMessageSession session)
-        {
-            heartbeatTimer?.Stop();
+            // don't block here since StartupTasks are executed synchronously.
+            _ = SendEndpointStartupMessage(DateTime.UtcNow, cancellationToken);
```
Comment (adamralph) [inline src/NServiceBus.Heartbeat/HeartbeatSender.cs:33]: I don't think we should pass the token here. `SendEndpointStartupMessage` runs recursively in a background task until it succeeds, after `OnStart` returns. If the token is cancelled after `OnStart` returns, `SendEndpointStartupMessage` must not be affected.

File: src/NServiceBus.Heartbeat/HeartbeatSender.cs [hunk 2]
```diff
@@ -23,41 +22,56 @@ class HeartbeatSender : FeatureStartupTask, IDisposable
...
+                        {
+                            // no-op
+                        }
+                        catch (Exception ex)
+                        {
+                            Logger.Warn("Unable to send heartbeat to ServiceControl.", ex);
+                        }
+                    }
+                },
+                cancellationToken);
```
Comment (adamralph) [inline src/NServiceBus.Heartbeat/HeartbeatSender.cs:65]: We shouldn't pass the token here unless we want to participate in cooperative cancellation in this method, in which case we need to do this — https://github.com/Particular/NServiceBus.RavenDB/blob/371154f9c9293edc728e9dc4b461739f37a4e6e3/src/NServiceBus.RavenDB/Outbox/RavenDbOutboxStorage.cs#L69-L72

File: src/NServiceBus.Heartbeat/HeartbeatSender.cs (hunk 1 above)
Comment (adamralph) [reply src/NServiceBus.Heartbeat/HeartbeatSender.cs:33]: Come to think of it... `OnStop()` should probably cancel that background task.

Comment (adamralph) [reply src/NServiceBus.Heartbeat/HeartbeatSender.cs:33]: Discussed with @DavidBoike and will make both changes.

File: src/NServiceBus.Heartbeat/HeartbeatSender.cs (hunk 2 above)
Comment (adamralph) [reply src/NServiceBus.Heartbeat/HeartbeatSender.cs:65]: Discussed with @DavidBoike and will change to `CancellationToken.None`.

**Pull request merged**